WordPressCom-Stats-iOS/UI/StatsTwoColumnTableViewCell.m

```diff
@@ -83,22 +68,22 @@ - (void)doneSettingProperties
         backgroundView.contentBackgroundView.backgroundColor = [WPStyleGuide statsNestedCellBackground];
     }
     
-    self.leftHandGlyphLabel.textColor = [WPStyleGuide grey];
     if (self.expandable && self.expanded) {
-        self.leftHandGlyphLabel.text = StatsTwoColumnCellChevronExpanded;
+        self.leftHandGlyph.image = [[UIImage imageNamed:@"icon-chevron-up-20x20" inBundle:self.statsBundle compatibleWithTraitCollection:nil] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
```

Comment:
Yeah I'm going to log this as another issue - definitely could use some improvement there. :smile: